CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Changed`
 
 - [#296](https://github.com/nf-core/epitopeprediction/pull/296) Major rework of Fasta output: new header structure, full wildtype sequences, mutated peptides trimmed around mutation site in a flanking region of defined size and support for overlapping mutations.
+- [#310](https://github.com/nf-core/epitopeprediction/pull/310) Drop conda support temporarily until conda lock file support is implemented
```

Comment:
are lock files really the problem here? isn't it more that the underlying packages don't support modern python?